### PR TITLE
Add Browserify

### PR DESCRIPTION
**Pull request by YerkoPalma**
**https://github.com/ungoldman/awesome-browserify**

**A curated list of awesome Browserify resources, libraries, and tools.**

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Feedback:

- The `Contents` section should be `##`
- Drop all the `A` / `An` prefixes in the descriptions.
- You should include at least one demo in the `Demos` section.
- I'd like to see more articles in the `Articles` section. You could pick the best ones from: http://browserify.org/articles.html
- Include some links to community resources (Stack Overflow tag, IRC, chat, forums, communities. Example: https://github.com/sindresorhus/awesome-electron#community)
- Maybe include some videos, talks, etc, on YouTube.

File: readme.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [choo](https://github.com/YerkoPalma/awesome-choo) - App framework.
 - [Redux](https://github.com/brillout/awesome-redux) - State container for JavaScript apps.
 - [webpack](https://github.com/d3viant0ne/awesome-webpack) - Module bundler.
+- [Browserify](https://github.com/ungoldman/awesome-browserify) - browser-side require().
```
Comment (sindresorhus) [inline readme.md]: Better description:

> Module bundler.

**Comment by YerkoPalma** [comment]
Thanks for the feedback, I'll ping back when it gets fixed

**Comment by YerkoPalma** [comment]
@sindresorhus the list and the PR are updated. Please check again :)

**Pull request merged**